README.md

```diff
@@ -1,5 +1,7 @@
 # HelenOS
 
+![screenshot](http://www.helenos.org/chrome/site/screenshot.png "Screenshot")
```

Comment:
Maybe the following URL is slightly better: http://www.helenos.org/raw-attachment/wiki/Screenshots/screenshot.png

Reply:
Isn't it possible to display image from the git repo itself?

Reply:
It is. Where do I put the image in the repo? Should I create a resources/assets directory in the repo and put the image there?

Reply:
There's some non-code stuff in `contrib/`, so maybe there? @jermar?

Reply:
TBH, I am not a big fan of including screenshots into the repository.

Reply:
Ok, an external link is fine then.